docs/index.rst

```diff
@@ -1,27 +1,18 @@
-.. edx-developer-docs documentation master file, created by
-   sphinx-quickstart on Wed Nov 28 11:51:14 2018.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+#############################
+Developer Documentation Index
+#############################
 
-edx-developer-docs
-===============================
-Developer Documentation
+This the landing page for all documentation relevant to developers of the Open
+edX platform.
 
-Contents:
+.. list-table::
```

Comment:
Were you planning on adding a link to private docs, or not until you know where you want to point to?

Reply:
I still think it might be better to keep the link to private docs in a private place.  It isn't too much to ask edX engineers to know two places: the common open place with all public info, and the internal place that has the private information.

Reply:
@nedbat: http://docs.edx.org/ is an official landing page.  I want a more official landing page for these other docs so it isn't the same as today where there are potentially many landing pages.  For example, Andy and I had one in Confluence.  @nasthagiri thought of the onboarding docs as a more official landing place.  Not sure if others have their own, or aren't aware of any.

I think it is hard to have an official starting place from inside Confluence.  Do we need/want a separate domain like http://docs-internal.edx.org/ that can in turn point to docs.edx.org where needed?  Or, we could add a link to docs.edx.org for edX employees that could point to a page that breaks down our docs however we want?  Or, we could this same link, but word it more like someone might be lost: "If you meant to find edx.org internal documentation, please go here instead."

I'm open to other ideas as well.  I just want a discussion of what we should do, and not what we shouldn't do.  Thanks.

Reply:
It is true that docs.edx.org has books that are only about edX.  But the audience for those books are the general public, so it makes sense to have them visible on a public domain.

These "internal-only" developer docs are only intended for edX engineers.  We have an entire wiki full of important information, and we expect edX engineers to know how to navigate that wiki.  I don't understand why edX engineers need a top-level domain to find these docs.

Reply:
1. I am not proposing another top-level domain. I'm arguing that it should be unnecessary.
2. A more-curated page in the wiki sounds great.
3. That page on the wiki can then point to these public docs at the very top.

Reply:
My point #1 was meant to agree with you. I made that more clear. 

We’ll start with Confluence, and see how we can both make that page more useful and get better publicity for it, both of which will complement one another.

Reply:
Thank you for this discussion.

Here's the landing page for edX internal engineers: https://openedx.atlassian.net/wiki/spaces/ENG/overview
It will include a link to this Developer Doc Index page once it is published.  I am removing the `edX.org Development` section from this doc.  This means edX internal engineers have only one landing page: the confluence page above.

Reply:
OK, cool :)